airflow/dag_processing/manager.py

```diff
@@ -530,10 +535,7 @@ def _run_parsing_loop(self):
         while True:
             loop_start_time = time.monotonic()
             ready = multiprocessing.connection.wait(self.waitables.keys(), timeout=poll_time)
-            if self._signal_conn in ready:
-                if conf.getboolean("scheduler", "standalone_dag_processor"):
-                    # Nothing to do if callbacks are not stored in the database
-                    self._fetch_callbacks(maxCallbacksPerLoop)
+            if self._signal_conn in ready and not self._standalone_mode:
```

Comment:
Same as above. This if can be simplified I think:

```
            if self._standalone_mode:
                self._fetch_callbacks(max_callbacks_per_loop)
            else:
                ready = multiprocessing.connection.wait(self.waitables.keys(), timeout=poll_time)
                if self._signal_conn in ready:
````

I think there is no point in cheacking "_signal_conn" and "ready" in standaline mode.